Patches/WWII Soviet Faction/WWIISovietFaction_CE_Patch_Clothing.xml

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationSequence">
+		<success>Always</success>
+		<operations>
+
+			<li Class="CombatExtended.PatchOperationFindMod">
+				<modName>WWII Soviet Faction (1.0)</modName>
+			</li>
+
+			<!-- ========== M73 Uniform ========== -->
+
+			<li Class="PatchOperationAdd">
+				<xpath>Defs/ThingDef[defName="Apparel_M73_Uniform"]/statBases</xpath>
+				<value>
+					<Bulk>8</Bulk>
+					<WornBulk>4</WornBulk>
```

Comment:
This is just the summer uniform, isn't it? Is the top being treated like a proper jacket or just a long sleeve shirt? If the former, you're probably more or less alright. If the latter, the bulk should probably be set to 2.

In base CE for instance, the combined wornBulk for the vanilla Jacket + Pants + T-Shirt would only be 3.

Reply:
Since the original mod treats this as a complete uniform (with pants), that make it comparable to your vanilla Jacket + Pants + T-Shirt example, so it'd probably make more sense to lower `WornBulk` down to 3 instead.

Reply:
Fixed in e7410ab